### PR TITLE
ENG-3802: change widget config page layout

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Review by ichalagashvili — DISMISSED** [review]
LGTM

File: src/locales/en.js [hunk 1]
```diff
@@ -318,7 +318,7 @@ export default {
     'menu.details': 'Details',
     'menu.groups': 'Groups',
     'menu.database': 'Database',
-    'menu.widget': 'MFE & Widgets',
+    'menu.widget': 'Widgets',
```
Comment (Dv92) [inline src/locales/en.js]: this value should remain 'MFE & Widgets'

File: src/locales/it.js [hunk 2]
```diff
@@ -319,7 +319,7 @@ export default {
     'menu.details': 'Dettagli',
     'menu.groups': 'Gruppi',
     'menu.database': 'Database',
-    'menu.widget': 'MFE & Widgets',
+    'menu.widget': 'Widgets',
```
Comment (Dv92) [inline src/locales/it.js]: same

File: src/locales/pt.js [hunk 3]
```diff
@@ -312,7 +312,7 @@ export default {
     'menu.details': 'Detalhes',
     'menu.groups': 'Grupos',
     'menu.database': 'Banco de Dados',
-    'menu.widget': 'MFE & Widgets',
+    'menu.widget': 'Widgets',
```
Comment (Dv92) [inline src/locales/pt.js]: same

File: src/locales/en.js (hunk 1 above)
Comment (gReis89) [reply src/locales/en.js]: The designs shows only Widgets, but I'll roll it back

**Review by ichalagashvili — DISMISSED** [review]

**Review by Dv92 — DISMISSED** [review]

**Review by ichalagashvili — APPROVED** [review]

**Pull request merged**